src/pages/Regions/Region.tsx

```diff
@@ -373,7 +454,20 @@ const Region = () => {
       }
     }
     else{
-      await editRegionAdmin(admin);
+      if(admin.adminType.adminTypeName === "Голова ОПР" ||
+      admin.adminType.adminTypeName === "Член ОПР" ||
+      admin.adminType.adminTypeName === Roles.OkrugaHead ||
+      admin.adminType.adminTypeName === Roles.OkrugaHeadDeputy){
+        if(await getCheckPlastMember(admin.userId)){
+          await editRegionAdmin(admin);
+        }
+        else {
+          showPlastMemberDisable(admin);
+        }
+      }
+      else{
+        await editRegionAdmin(admin);
+      }
```

Comment:
I think, we can simplify this construction by using 
457 - else if (this big check) {
  here are this if/else statement from 461-467
 }
else - from lines 468-470
However, it's on you